jsk_perception/node_scripts/alex_object_recognition.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python
```

Comment:
I think `AlexNet` is the common name, so please rename this to `alexnet_object_recognition.py`.